Source/Core/VideoCommon/UberShaderPixel.cpp

```diff
@@ -1133,7 +1126,7 @@ ShaderCode GenPixelShader(APIType api_type, const ShaderHostConfig& host_config,
               "  }}\n");
   }
 
-  if (use_shader_blend)
+  if (use_framebuffer_fetch)
```

Comment:
This seems incorrect (or unfinished) as there are two blocks of `use_framebuffer_fetch`

Reply:
Yeah, the second one is now unnecessary.  Removed.